src/model.js

```diff
@@ -2571,7 +2571,13 @@ Specify a different name for either index to resolve this issue.`);
       }
     }
 
-    const instances = records.map(values => this.build(values, { isNewRecord: true, include: options.include }));
+    const instances = records.map(values => this.build(values, {
+      isNewRecord: true,
+      attributes: options.fields,
+      include: options.include,
+      raw: options.raw,
+      silent: options.silent,
```

Comment:
Can you add a test for `silent`?

Reply:
Sorry, `silent` is not a valid missing option, I notice the test for silent is already passed in integration test.